app/src/services/census/CensusStreamService.ts

```diff
@@ -11,6 +11,8 @@ import Census from '../../config/census';
 
 @injectable()
 export default class CensusStreamService implements ServiceInterface {
+    public readonly bootPriority = 10;
```

Comment:
I'd say this should be 11, we want the database booted before Census socket

Reply:
Not really as the Census socket doesn't use the boot method. It starts in the start method. Mostly the use service that depend on each other when booting(e.g. booting a service that requires a database connection).

Reply:
Ok, don't we then want this implemented on the start level as well?

Reply:
Don't think anything should depend on each other when starting. Basically you prepare the app in boot, and then give the start signal with start.

Reply:
Fair enough